tests/phpunit/tests/REST_API/Fonts_Controller.php

```diff
@@ -33,9 +33,12 @@ public function test_register_routes() {
 	}
 
 	public function test_get_item_schema() {
-		$request    = new WP_REST_Request( 'OPTIONS', '/amp/v1/fonts' );
-		$response   = rest_get_server()->dispatch( $request );
-		$data       = $response->get_data();
+		$request  = new WP_REST_Request( 'OPTIONS', '/amp/v1/fonts' );
+		$response = rest_get_server()->dispatch( $request );
+		$data     = $response->get_data();
+
+		$this->assertNotEmpty( $data );
```

Comment:
Why is this being added here?

Reply:
In case there is an issue with the REST API routes (e.g. they aren't properly registered, which happened during refactoring), this will fail the assertion instead of causing PHP warnings because `$data` is empty. Just a nicer way to fail the tests in those cases.